test/deploy/linux/cassandra/install/debian/roles/prepare/tasks/installCassandra.yml

```diff
@@ -7,7 +7,7 @@
   become: true
 
 - name: add cassandra keys
-  shell: wget -q -O - https://www.apache.org/dist/cassandra/KEYS | sudo apt-key add -
+  shell: wget -q -O - https://downloads.apache.org/cassandra/KEYS | sudo apt-key add -
```

Comment:
To fix:

<img width="1446" alt="image" src="https://github.com/newrelic/open-install-library/assets/102765645/d37908cc-1118-44f2-b946-ad0dfd867d7f">

Reply:
The **_wrong_** KEYS file URL should be the one causing the unit test validation to fail.